Sources/FoundationInternationalization/TimeZone/TimeZone_ICU.swift

```diff
@@ -309,6 +316,23 @@ internal final class _TimeZoneICU: _TimeZoneProtocol, Sendable {
         return result
     }
 
+    #if os(Windows)
+    internal static func getSystemTimeZoneID(forWindowsIdentifier identifier: String) -> String? {
+        let timeZoneIdentifier = Array(identifier.utf16)
+        let result: String? = timeZoneIdentifier.withUnsafeBufferPointer { identifier in
+            return _withResizingUCharBuffer { buffer, size, status in
+                let len = ucal_getTimeZoneIDForWindowsID(identifier.baseAddress, Int32(identifier.count), nil, buffer, size, &status)
+                if status.isSuccess {
```

Comment:
return status.isSuccess ? len : nil feels like a better way to write this.

Reply:
Yeah that seems fine too, I was just copying the format of the above `getCanonicalTimeZoneID` function for consistency